public/app/features/dashboard/components/GenAI/GenAIPanelDescriptionButton.tsx

```diff
@@ -39,7 +43,9 @@ function getMessages(panel: Panel): Message[] {
 
   return [
     {
-      content: DESCRIPTION_GENERATION_STANDARD_PROMPT,
+      content:
+        `${DESCRIPTION_GENERATION_STANDARD_PROMPT}\n` +
+        `Disregard the current panel description and come up with one that makes sense given the panel's type and purpose. The panel's type is ${panel.type}`,
```

Comment:
Given the comments [here](https://github.com/grafana/grafana/pull/88429/files#r1619294174), I'm wondering if we should have "the disregard the current panel" prompt set to a user role. The same applies to the title.